env.sh

```diff
@@ -53,12 +64,21 @@ if ! ${PYTHON} --version; then
     export PYTHON=python2.7
     if ! ${PYTHON} --version; then
         echo "Error: python is not found"
-        exit
+        exit 1
     fi
 fi
 
-# set DORIS_THIRDPARTY
-if [ -z ${DORIS_THIRDPARTY} ]; then
-    export DORIS_THIRDPARTY=${DORIS_HOME}/thirdparty
+# set GCC HOME
+if [[ -z ${DORIS_GCC_HOME} ]]; then
+    # specify your GCC HOME if needed
+    export DORIS_GCC_HOME=$(dirname `which gcc`)/..
+fi
+
+gcc_ver=`${DORIS_GCC_HOME}/bin/gcc -dumpversion`
+required_ver="5.3.1"
+if [[ ! "$(printf '%s\n' "$required_ver" "$gcc_ver" | sort -V | head -n1)" = "$required_ver" ]]; then 
+    echo "Error: GCC version (${gcc_ver}) must be greater than or equal to ${required_ver}"
+    echo "DORIS_GCC_HOME is need to be set in env.sh after you installed a new version of GCC."
```

Comment:
I think this prompt is not good, we should guide user to set environment variable DORIS_GCC_HOME to its available gcc

Reply:
OK